### PR TITLE
Minor refactor and Partial Get Example

### PR DESCRIPTION
**Pull request by rpmoore**

### CODE REVIEW

**Comment by RachelTucker** [comment]
The branch is out-of-date

File: ds3-sdk-integration/src/main/java/com/spectralogic/ds3client/integration/Util.java [hunk 1]
```diff
@@ -74,8 +75,8 @@ public static void loadBookTestData(final Ds3Client client, final String bucketN
         final List<Ds3Object> objects = new ArrayList<>();
 
         for(final String book : BOOKS) {
-            final File objFile = ResourceUtils.loadFileResource(resourceBaseName + book);
-            final Ds3Object obj = new Ds3Object(book, objFile.length());
+            final Path objFile = ResourceUtils.loadFileResource(resourceBaseName + book);
```
Comment (RachelTucker) [inline ds3-sdk-integration/src/main/java/com/spectralogic/ds3client/integration/Util.java]: Is there a reason why this is called objFile when it is now a Path object?

File: ds3-sdk-samples/src/main/java/com/spectralogic/ds3client/samples/PartialObjectGetExample.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
+package com.spectralogic.ds3client.samples;
```
Comment (RachelTucker) [inline ds3-sdk-samples/src/main/java/com/spectralogic/ds3client/samples/PartialObjectGetExample.java]: Missing copyright

File: ds3-sdk-integration/src/main/java/com/spectralogic/ds3client/integration/Util.java (hunk 1 above)
Comment (rpmoore) [inline ds3-sdk-integration/src/main/java/com/spectralogic/ds3client/integration/Util.java]: no?...

File: ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/DataIntegrity_Test.java [hunk 3]
```diff
@@ -68,9 +66,9 @@ public void singleFilePut() throws IOException, URISyntaxException, XmlProcessin
         try {
             helpers.ensureBucketExists(bucketName);
 
-            final File objFile = ResourceUtils.loadFileResource(Util.RESOURCE_BASE_NAME + book);
-            final String digest = DigestUtils.sha256Hex(new FileInputStream(objFile));
-            final Ds3Object obj = new Ds3Object(book, objFile.length());
+            final Path objFile = ResourceUtils.loadFileResource(Util.RESOURCE_BASE_NAME + book);
```
Comment (RachelTucker) [inline ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/DataIntegrity_Test.java]: Path is named objFile

File: ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/Smoke_Test.java [hunk 4]
```diff
@@ -468,10 +467,10 @@ public void testRecoverWriteJob() throws SignatureException, IOException, XmlPro
             client.putBucket(new PutBucketRequest(bucketName));
             helpers.ensureBucketExists(bucketName);
 
-            final File objFile1 = ResourceUtils.loadFileResource(Util.RESOURCE_BASE_NAME + book1);
-            final File objFile2 = ResourceUtils.loadFileResource(Util.RESOURCE_BASE_NAME + book2);
-            final Ds3Object obj1 = new Ds3Object(book1, objFile1.length());
-            final Ds3Object obj2 = new Ds3Object(book2, objFile2.length());
+            final Path objFile1 = ResourceUtils.loadFileResource(Util.RESOURCE_BASE_NAME + book1);
```
Comment (RachelTucker) [inline ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/Smoke_Test.java]: Path is named objFile

File: ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/Smoke_Test.java [hunk 5]
```diff
@@ -568,10 +567,10 @@ public void testRecoverReadJob() throws SignatureException, IOException, XmlProc
         final String bucketName = "test_recover_read_job_bucket";
         final String book1 = "beowulf.txt";
         final String book2 = "ulysses.txt";
-        final File objFile1 = ResourceUtils.loadFileResource(Util.RESOURCE_BASE_NAME + book1);
-        final File objFile2 = ResourceUtils.loadFileResource(Util.RESOURCE_BASE_NAME + book2);
-        final Ds3Object obj1 = new Ds3Object(book1, objFile1.length());
-        final Ds3Object obj2 = new Ds3Object(book2, objFile2.length());
+        final Path objFile1 = ResourceUtils.loadFileResource(Util.RESOURCE_BASE_NAME + book1);
```
Comment (RachelTucker) [inline ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/Smoke_Test.java]: Path is named objFile

File: ds3-sdk/src/test/java/com/spectralogic/ds3client/Ds3Client_Test.java [hunk 6]
```diff
@@ -305,10 +306,10 @@ public void putObject() throws IOException, SignatureException, URISyntaxExcepti
         queryParams.put("job", jobIdString);
         queryParams.put("offset", Long.toString(0));
         
-        final File resourceFile = ResourceUtils.loadFileResource("LoremIpsumTwice.txt");
-        final byte[] fileBytes = Files.readAllBytes(resourceFile.toPath());
+        final Path resourceFile = ResourceUtils.loadFileResource("LoremIpsumTwice.txt");
```
Comment (RachelTucker) [inline ds3-sdk/src/test/java/com/spectralogic/ds3client/Ds3Client_Test.java]: Path object named resourceFile

**Comment by RachelTucker** [comment]
Looks good to me.  I can't merge it with it being out-of-date...

**Pull request merged**